fe/fe-core/src/main/java/com/starrocks/sql/ast/spm/ShowBaselinePlanStmt.java

```diff
@@ -37,8 +42,30 @@ public class ShowBaselinePlanStmt extends ShowStmt {
             .addColumn(new Column("updateTime", ScalarType.createVarchar(60)))
             .build();
 
-    public ShowBaselinePlanStmt(NodePosition pos) {
+    public static final Map<String, Type> BASELINE_FIELD_META = ImmutableMap.<String, Type>builder()
+            .put("id", Type.BIGINT)
+            .put("global", Type.BOOLEAN)
+            .put("enable", Type.BOOLEAN)
+            .put("bindsqldigest", Type.VARCHAR)
+            .put("bindsqlhash", Type.BIGINT)
+            .put("bindsql", Type.VARCHAR)
+            .put("plansql", Type.VARCHAR)
+            .put("costs", Type.DOUBLE)
+            .put("queryms", Type.DOUBLE)
+            .put("source", Type.VARCHAR)
+            .put("updatetime", Type.VARCHAR)
+            .build();
+
+    // save where clause
+    private final Expr where;
```

Comment:
I think we can reuse the `predicate` field of `ShowStmt`

Reply:
the restrictions of `prdicate` are too many, not friendly